extensions/sql-bindings/src/sql-bindings.d.ts

```diff
@@ -42,8 +43,11 @@ declare module 'sql-bindings' {
 		/**
 		 * Prompts the user to enter connection setting and updates it from AF project
 		 * @param projectUri Azure Function project uri
-		 */
-		promptAndUpdateConnectionStringSetting(projectUri: vscode.Uri | undefined): Promise<string | undefined>;
+		 * @param connectionInfo (optional) connection info from the user to update the connection string, 
+		 * if left undefined we prompt user to get connection info manually or list of connection profiles 
```

Comment:
I'd just leave out the `or list of connections profiles`. Not only does it not really read correctly, it's fine to just say `we prompt the user for the connection info`